src/markdown-pages/contribute-to-quickstarts/index.mdx

```diff
@@ -1,17 +1,81 @@
 ---
-title: 'Quickstarts'
-template: 'OverviewTemplate'
-description: ''
+title: 'Contribute a Quickstart to New Relic I/O'
+template: 'GuideTemplate'
+description: 'Quickstarts course'
 ---
 
 <Intro>
 
-New Relic provides different solutions for you to kick start your observability journey. 
-But wouldn't it be cool if you wouldn't have to manually set up the observability building block? Instead you could just click, install and get value from your data.
+Imagine you’ve developed a new database called FlashDB, the lightest and fastest database in space. Your database is already very popular among developers.
```

Comment:
Not sure how to reword this but 'the lightest and fastest database in space' doesn't seem right. Maybe the 'the lightest and fastest service in the database space' could work?